### ISSUE TITLE
SharePoint default Language

### ISSUE THREAD

**Issue by J-a-k-o-b** [OfficeDev/microsoft-teams-apps-requestateam#231]
Hi,
When cloning/creating a Team with the graph functions used in this solution, the new Team comes with an SharePoint which default language is set to the default tenant language.
Is anyone aware if there is an easier solution than pulling the site creation before the team-creation to be able to create a Team where the SPO language is set to a custom language?

**Comment by alexc-MSFT** [comment]
@J-a-k-o-b Sadly there is a limitation here that is not covered by the graph. The regional settings always inherit from the tenant and we cannot override these through the graph calls.

The only options I have found so far are - 

- Use SharePoint site designs
- Use client side object model (CSOM) code

The second option will require legacy auth to be turned on at the tenant level.

At the moment there are no plans to try and address this due to the fact we limited to only the above options

**Comment by J-a-k-o-b** [comment]
Yeah, i think i will completely redesign the LA here to use a PS Runbook which uses the PNP Module with an APP Registration to create a SPO TeamSite with a custom language and enable the Group afterwards for Teams. The configuration of the team has to be done than by Code and not with the clone feature.
Thank you so far. The Request a Team App is great!
